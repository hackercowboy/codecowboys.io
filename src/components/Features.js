import React from 'react';
import PropTypes from 'prop-types';

import './Features.scss';

function Features({ children }) {
  return (
    <div className="features">
      {children}
    </div>
  );
}

Features.propTypes = {
  children: PropTypes.node,
};

Features.defaultProps = {
  children: undefined,
};

export default Features;
