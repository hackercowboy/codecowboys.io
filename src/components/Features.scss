.features {
  @media only screen and (min-width : 992px) {
    align-items: center;
    display: flex;
  }
}
