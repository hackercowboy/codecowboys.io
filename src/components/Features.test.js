import React from 'react';
import { shallow } from 'enzyme';

import Features from './Features';

describe('<Features/>', () => {
  it('should render', () => {
    const wrapper = shallow(<Features />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
