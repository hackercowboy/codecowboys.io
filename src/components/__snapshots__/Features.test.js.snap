// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Features/> should render 1`] = `"<div className=\\"features\\" />"`;
